test/client/fixtures/sandbox/event/unload-test.js

```diff
@@ -30,6 +30,17 @@ asyncTest('BEFORE_UNLOAD_EVENT must be called last (GH-400)', function () {
         });
 });
 
+test('Hammerhead UNLOAD_EVENT must be called (GH-1095)', function () {
```

Comment:
Why do you add `(GH-1095)` suffix to the test name?
test name should be in lowercase.

Reply:
I thought, if you fix an issue and write a test for this case, you need to add such a suffix with issue number. Am i wrong?